Makefile

```diff
@@ -160,15 +160,17 @@ build: all
 goreleaser:
 	curl -sfL -o goreleaser-install https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh
 	sh goreleaser-install v0.135.0
+	go build -o $(GOPATH)/bin/pkg-config github.com/influxdata/pkg-config
+	install xcc.sh $(GOPATH)/bin/xcc
 
 # Parallelism for goreleaser must be set to 1 so it doesn't
 # attempt to invoke pkg-config, which invokes cargo,
 # for multiple targets at the same time.
 dist: goreleaser
-	./bin/goreleaser -p 1 --rm-dist --config=.goreleaser-nightly.yml
+	./bin/goreleaser -p 1 --skip-validate --rm-dist --config=.goreleaser-nightly.yml
 
 nightly: goreleaser
-	./bin/goreleaser -p 1 --rm-dist --config=.goreleaser-nightly.yml
+	./bin/goreleaser -p 1 --skip-validate --rm-dist --config=.goreleaser-nightly.yml
```

Comment:
Why `--skip-validate`?

Reply:
This is all `--snapshot` does. As we're not using snapshot, because we can't publish the artifacts, we can use `--skip-validate` to by-pass anyway. It skips tests that there are no new commits in the working tree since the previous tag was cut.

https://github.com/goreleaser/goreleaser/blob/983f342ab8501a9c7f1ca699b05ee2817e93b9b3/cmd/build.go#L99